src/Error/Renderer/ConsoleErrorRenderer.php

```diff
@@ -16,23 +16,38 @@
  */
 namespace Cake\Error\Renderer;
 
+use Cake\Console\ConsoleOutput;
 use Cake\Error\ErrorRendererInterface;
 use Cake\Error\PhpError;
 
 /**
  * Plain text error rendering with a stack trace.
  *
- * Writes to STDERR for console environments
+ * Writes to STDERR via a Cake\Console\ConsoleOutput instance for console environments
  */
 class ConsoleErrorRenderer implements ErrorRendererInterface
 {
+    /**
+     * @var \Cake\Console\ConsoleOutput
+     */
+    private $output;
```

Comment:
The other private var was changed it protected in 5.x. We keep it consistent here or there.

https://github.com/cakephp/cakephp/blob/5.x/src/Error/Renderer/TextExceptionRenderer.php#L33